pkg/chunk/schema_config.go

```diff
@@ -144,6 +145,22 @@ func (cfg *SchemaConfig) ForEachAfter(t model.Time, f func(config *PeriodConfig)
 	}
 }
 
+func validateChunks(cfg PeriodConfig) error {
+	objectStore := cfg.IndexType
+	if cfg.ObjectType != "" {
+		objectStore = cfg.ObjectType
+	}
+	switch objectStore {
+	case "cassandra", "aws-dynamo", "bigtable-hashed", "gcp", "gcp-columnkey", "bigtable", "boltdb", "grpc-store":
```

Comment:
`"boltdb"`  is not an option for the chunk store. Should be removed from this.

Reply:
Could you remove `boltdb`, please?

Reply:
My apologies, I removed the tests for BoltDb but forgot to remove it from the check list.